### ISSUE TITLE
HassSetVolume does not work for media players in idle state

### ISSUE THREAD

**Issue by mitrokun** [home-assistant/intents#2782]
Has anything changed under the hood? I just discovered that it is impossible to change the volume of an idle device.
As far as I remember, there was no such restriction before, _but I could be wrong._

In any case, it should be possible to change the volume, at least using the device name. 
Hearing  an error response after this simple request is very unexpected.

**Comment by andreasbrett** [comment]
True. Especially for AV receivers that are exposed as media_player but are always Idle because the are not playing back the media but rather are mere volume controllers. Yes, you could use the outdated universal media player integration but that thing is so old and still yaml...

**Comment by jackwilsdon** [comment]
This issue probably belongs in core. It was changed in https://github.com/home-assistant/core/pull/115445 (which also has a bit of discussion on it), the relevant code is here: 

https://github.com/home-assistant/core/blob/d676169b04cb26c76b3969c3f695e1a981062897/homeassistant/components/media_player/intent.py#L96-L97
